app/models/link.rb

```diff
@@ -100,8 +100,17 @@ def readonly?
     true
   end
 
-  # Refresh the view, concurrently if it has already been populated
+  # Refresh the view in a background job
   def self.refresh
+    if Rails.env.test?
+      refresh_materialized_view
```

Comment:
I would avoid the environment checking here and just use `:run_jobs` in any tests where you need the results of the job immediately.

Reply:
good suggestion. I was thinking this might be common in our test suit since we create a lot of relationships.

Reply:
This is what I use for handling jobs in RSpec: